tests/integration/components/o-s-s/information-section-test.ts

```diff
@@ -0,0 +1,109 @@
+      await render(hbs`<OSS::InformationSection @title={{this.title}} @plain={{false}}/>`);
+
+      assert.dom("[data-control-name='information-section--header']").hasClass('background-color-gray-50');
+    });
+
+    test('it renders the header with a white background when @plain is true', async function (assert) {
+      await render(hbs`<OSS::InformationSection @title={{this.title}} @plain={{true}}/>`);
+
+      assert.dom("[data-control-name='information-section--header']").hasClass('background-color-white');
+    });
+  });
+
+  module('badge icons', function () {
+    test('it renders the icon with @badgeIcon arg', async function (assert) {
+      await render(hbs`<OSS::InformationSection @badgeIcon={{this.icon}} @title={{this.title}}/>`);
+
+      assert.dom('.upf-badge i').hasClass(this.icon);
+    });
+
+    test('it renders the icon with an svg icon as @badgeIcon arg', async function (assert) {
+      await render(hbs`<OSS::InformationSection @svgIcon={{this.svgIcon}} @title={{this.title}}/>`);
+      assert.dom('.upf-badge img').hasAttribute('src', this.svgIcon);
+    });
+
+    test('it renders the badge with badge named block', async function (assert) {
+      await render(hbs` <OSS::InformationSection @title={{this.title}}>
+                          <:badge>
+                            <OSS::Badge
+                              data-control-name="test-badge"
+                              @icon={{this.icon}}
+                            />
+                          </:badge>
+                        </OSS::InformationSection>`);
+
+      assert.dom("[data-control-name='test-badge']").exists();
+    });
+  });
+
+  module('actions named block', function () {
+    test('it renders the action named block', async function (assert) {
+      await render(hbs` <OSS::InformationSection @title={{this.title}}>
+                          <:action>
+                            <div data-control-name="actions-named-block">
+                              <OSS::Button
+                                data-control-name="test-button"
+                                @label={{this.actionButton.label}}
+                                @icon={{this.actionButton.icon}}
+                              />
+                            </div>
+                          </:action>
+                        </OSS::InformationSection>`);
+
+      assert.dom("[data-control-name='actions-named-block']").exists();
+    });
+  });
+
+  module('content named block', function () {
+    test('it renders the badge with content named block', async function (assert) {
+      await render(hbs` <OSS::InformationSection @title={{this.title}}>
+                          <:content>
+                          <p data-control-name='content-named-block'>{{this.namedBlockContent}}</p>
+                          </:content>
+                        </OSS::InformationSection>`);
```

Comment:
nit: alignment of the <p> tag
It also looks like you have an empty state after each `await render` statement in some of the tests of the file

Reply:
Fixed testfile